vm/src/obj/objcomplex.rs

```diff
@@ -204,7 +204,11 @@ impl PyComplex {
         if re == 0.0 {
             format!("{}j", im)
         } else {
-            format!("({}+{}j)", re, im)
+            if im < 0.0 {
```

Comment:
Maybe sign forcing is possible? `format!("({}{:+}j)", re, im)`

Reply:
@youknowone Thanks for the comment. It works well